test/test_core.py

```diff
@@ -5597,24 +5597,14 @@ def test_utf8_textdecoder(self):
     self.do_runf('benchmark/benchmark_utf8.c', 'OK.')
 
   # Test that invalid character in UTF8 does not cause decoding to crash.
+  @also_with_minimal_runtime
   @parameterized({
-    '': [[]],
-    'textdecoder': [['-sTEXTDECODER']],
+    '': ([],),
+    'textdecoder': (['-sTEXTDECODER'],),
   })
   def test_utf8_invalid(self, args):
     self.do_runf('test_utf8_invalid.c', 'OK.', emcc_args=args)
 
-  # Test that invalid character in UTF8 does not cause decoding to crash.
```

Comment:
It looks like a few tests are removed. Is that suppose to be in this PR?

Reply:
The tests that were removed were just dupliates of existing tests that ran with `MINIMAL_RUNTIME`.   Instead if duplicating those tests I added `@also_with_minimal_runtime` which basically does the same thing (i.e. creates a duplicate of the test).